focuspoints.lrdevplugin/DefaultDelegates.lua

```diff
@@ -24,6 +24,7 @@ require "Utils"
 
 DefaultDelegates = {}
 DefaultDelegates.focusPointsMap = nil
+DefaultDelegates.focusPointDimen = nil
```

Comment:
yeah. I saw you added it back in the PointsRedererFactory. No idea why it is not on GitHub but in my code...